partcad/src/partcad/wrappers/wrapper_brep.py

```
@@ -0,0 +1,42 @@
+import wrapper_common
+
+
+def process(path, request):
+    try:
+        shape = TopoDS_Shape()
+        builder = BRep_Builder()
+        brep_tools = BRepTools()
+
+        if not brep_tools.Read_s(shape, path, builder):
+            raise Exception("BREP file could not be loaded")
+    except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)